server/sonar-server/src/main/java/org/sonar/server/plugins/edition/EditionInstaller.java

```diff
@@ -75,6 +76,12 @@ public boolean install(Set<String> editionPluginKeys) {
     }
   }
 
+  public void uninstall() {
+    Map<String, PluginInfo> pluginInfosByKeys = pluginRepository.getPluginInfosByKeys();
```

Comment:
I don't understand why you don't read and depend on the current license here

Reply:
no need

Reply:
true